### DISCUSSION TITLE
Elks word processor

### DISCUSSION THREAD

**Discussion by parabyte** [jbruchon/elks#2135] [General]
is this suitable for elks? 

[https://github.com/taviso/wpunix](url)

**Comment by ghaerr** [comment]
That's a pretty nice rendition of the old WordStar, very nice looking!

Unfortunately, it seems the project you mention is a set of scripts and patches to get the full Corel binary UNIX version of WordStar running on modern Linux; it's way too big for ELKS. This can be seen by the requirement for starting with the 32-bit version of WordStar from the COREL .iso 32-bit Linux binary distribution image.

**Comment by parabyte** [comment]
okay what about this?
[https://github.com/sleaze/WordTsar/tree/master](url)

**Comment by parabyte** [comment]
i can answer this myself wordtsar relies on QT :( sigh

**Reply by ghaerr** [reply]
And for those that remember using WordStar and know QT, realizing that one may now need QT in order to implement WordStar kind of blows one's mind...
